app/controllers/api/episodes_controller.rb

```diff
@@ -3,11 +3,12 @@ class Api::EpisodesController < Api::BaseController
 
   api_versions :v1
   represent_with Api::EpisodeRepresenter
-  find_method :find_by_guid
   filter_resources_by :podcast_id
 
+  find_method :find_by_guid
   after_action :process_media, only: [:create, :update]
   after_action :publish, only: [:create, :update, :destroy]
+  allow_params :show, [:id, :podcast_id, :guid_resource, :api_version, :format, :zoom]
```

Comment:
Are these not additive to the `allow_params` already in `Api::BaseController`?

I guess maybe it just [replaces the params](https://github.com/PRX/hal_api-rails/blob/044e132a89f95a72d2f55a79f335547830e46d4b/lib/hal_api/controller/actions.rb#L106) for the action every time you call it.